src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -690,6 +714,12 @@ ul {
   color: $gray7;
 }
 
+// Used for title/placeholder text of input fields.  This was required as some input fields require
+// were not using the expected color of $gray7
+::v-deep label,.v-label,theme--light {
```

Comment:
Check classes again.

Reply:
updated to use correct syntax and tested to be applying styles correctly.